linera-storage/src/db_storage.rs

```diff
@@ -432,6 +454,23 @@ where
         Ok(test)
     }
 
+    async fn contains_hashed_certificate_values(
+        &self,
+        hashes: Vec<CryptoHash>,
+    ) -> Result<Vec<bool>, ViewError> {
+        let mut keys = Vec::new();
+        for hash in hashes {
+            let value_key = bcs::to_bytes(&BaseKey::Value(hash))?;
```

Comment:
(I guess we should rename `BaseKey::Value` to `CertificateValue`.)

Reply:
Maybe another PR?